### PR TITLE
Implement to control playback rate on media remote

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch implements playback rate to control playback rate on media
remote.

Refs: #889

### CODE REVIEW

**Comment by jdragonbae** [comment]
@romandev , @codeimpl This is draft PR. Please, take a look if it is in the right way.

File: modules/public/mojom/media.mojom [hunk 1]
```diff
@@ -18,11 +18,13 @@ interface BananaMediaEventDispatcher {
   OnChangedPIPMode(bool value);
   OnEnteredVideoFullscreen(pending_remote<BananaMediaCommandProcessor> impl, bool isDownloadable);
   OnExitedVideoFullscreen();
+  OnPlaybackRateChanged(double rate);
```
Comment (jdragonbae) [inline modules/public/mojom/media.mojom:21]: Q. Do really need OnPlaybackRateChanged? Any other ideas?

Comment (romandev) [inline modules/public/mojom/media.mojom:21]: Could you please explain why we should implement this event?

Comment (romandev) [reply modules/public/mojom/media.mojom:21]: I thought that the event might be used for UI update, right?
I imagine that the we provide "playback rate list" on UI as follows:
  - 4x
  - 2x
  - 1x
  - 1/2x
  - 1/4x
If the video playback rate was changed to 0.01 (1 / 10), how can we update the UI?
(Other option might be that we implement slider UI)

Comment (jdragonbae) [reply modules/public/mojom/media.mojom:21]: > I thought that the event might be used for UI update, right?

Right.

> I imagine that the we provide "playback rate list" on UI as follows:
> 
> 4x
> 2x
> 1x
> 1/2x
> 1/4x

It is the same as I thought. Similar to youtube quality.

> If the video playback rate was changed to 0.01 (1 / 10), how can we update the UI?
> Other option might be that we implement slider UI)

My idea now is to adjust the value. But, I'm not sure this way would be good.
Also, slider UI seems to be fine.

Comment (codeimpl) [reply modules/public/mojom/media.mojom:21]: We think get() api is better than event handler.

**Review by romandev — COMMENTED** [review]
As I remember, we decided to have a getter for playback rate, didn't it?

**Review by romandev — APPROVED** [review]
LGTM

**Pull request merged**